arm-datalake-analytics/job/2015-11-01-preview/swagger/job.json

```diff
@@ -585,21 +586,25 @@
         },
         "totalCompilationTime": {
           "type": "string",
+          "format": "non-iso-duration",
```

Comment:
Does swagger support this kind of format? https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types 

Does AutoRest support this kind of format ?